### ISSUE TITLE
Compilation issue instance_statistics C++

### ISSUE THREAD

**Issue by ManuelJNunez** [rticommunity/rticonnextdds-examples#387]
### Information

- **RTI Product**: Connext DDS
- **Version**: 6.1.0
- **Operating system**: Linux
- **Compiler**: GCC
- **Compiler version**: 7.5.0
- **Additional information**:

### What is the current behavior?

The example cannot compile:

```plaintext
[ 11%] Built target instance_statistics_cpp_qos
[ 11%] Built target instance_statistics_CXX_sources
Consolidate compiler generated dependencies of target instance_statistics_CXX_obj
[ 55%] Built target instance_statistics_CXX_obj
Consolidate compiler generated dependencies of target instance_statistics_subscriber_cpp
[ 77%] Built target instance_statistics_subscriber_cpp
Consolidate compiler generated dependencies of target instance_statistics_publisher_cpp
[ 88%] Building CXX object CMakeFiles/instance_statistics_publisher_cpp.dir/instance_publisher.cxx.o
/home/mnunez/rticonnextdds-examples/examples/connext_dds/instance_statistics/c++/instance_publisher.cxx:149:5: error: ‘retcode’ does not name a type; did you mean ‘revoke’?
     retcode = instanceTypeSupport::delete_data(data);
     ^~~~~~~
     revoke
/home/mnunez/rticonnextdds-examples/examples/connext_dds/instance_statistics/c++/instance_publisher.cxx:150:5: error: expected unqualified-id before ‘if’
     if (retcode != DDS_RETCODE_OK) {
     ^~
/home/mnunez/rticonnextdds-examples/examples/connext_dds/instance_statistics/c++/instance_publisher.cxx:156:5: error: expected unqualified-id before ‘return’
     return shutdown_participant(participant, "Shutting down", EXIT_SUCCESS);
     ^~~~~~
/home/mnunez/rticonnextdds-examples/examples/connext_dds/instance_statistics/c++/instance_publisher.cxx:157:1: error: expected declaration before ‘}’ token
 }
 ^
/home/mnunez/rticonnextdds-examples/examples/connext_dds/instance_statistics/c++/instance_publisher.cxx:24:12: warning: ‘int shutdown_participant(DDSDomainParticipant*, const char*, int)’ used but never defined
 static int shutdown_participant(
            ^~~~~~~~~~~~~~~~~~~~
CMakeFiles/instance_statistics_publisher_cpp.dir/build.make:75: recipe for target 'CMakeFiles/instance_statistics_publisher_cpp.dir/instance_publisher.cxx.o' failed
make[2]: *** [CMakeFiles/instance_statistics_publisher_cpp.dir/instance_publisher.cxx.o] Error 1
CMakeFiles/Makefile2:168: recipe for target 'CMakeFiles/instance_statistics_publisher_cpp.dir/all' failed
make[1]: *** [CMakeFiles/instance_statistics_publisher_cpp.dir/all] Error 2
Makefile:90: recipe for target 'all' failed
make: *** [all] Error 2
```

### Steps to reproduce the issue

Compile the example instance_statistics C++

### Suggested solutions

Remove the extra `}` causing a syntax problem.

**Comment by ManuelJNunez** [comment]
This issue is solved and will close once the commit is merged into master.
